receiver/windowsservicereceiver/winapi.go

```diff
@@ -0,0 +1,51 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+//go:build windows
+
+package windowsservicereceiver
+
+import (
+	"syscall"
+
+	"golang.org/x/sys/windows"
+	"golang.org/x/sys/windows/svc/mgr"
+)
+
+type Manager struct {
+	Handle windows.Handle
+}
+
+func SCConnect() (*Manager, error) {
+	var s *uint16
+
+	h, err := windows.OpenSCManager(s, nil, windows.GENERIC_READ)
+	if err != nil {
+		return nil, err
+	}
+
+	if err != nil {
+		return nil, err
+	}
+
+	return &Manager{
+		Handle: h,
+	}, nil
+}
+
+func (m *Manager) Disconnect() error {
+	return windows.CloseServiceHandle(m.Handle)
+}
+
+func (m *Manager) OpenService(sName string) (*mgr.Service, error) {
```

Comment:
nit: 
```suggestion
func (m *Manager) OpenService(name string) (*mgr.Service, error) {
```

The name of the function makes clear that the name refers to the service name.